base/linalg/rowvector.jl

```diff
@@ -73,18 +74,34 @@ julia> transpose(v)
 ```
 """
 @inline transpose(vec::AbstractVector) = RowVector(vec)
-@inline ctranspose{T}(vec::AbstractVector{T}) = RowVector(conj(vec))
+@inline ctranspose{T}(vec::AbstractVector{T}) = RowVector(_conj(vec))
 @inline ctranspose{T<:Real}(vec::AbstractVector{T}) = RowVector(vec)
 
 @inline transpose(rowvec::RowVector) = rowvec.vec
+@inline transpose(rowvec::ConjRowVector) = copy(rowvec.vec) # remove the ConjArray wrapper from any raw vector
 @inline ctranspose{T}(rowvec::RowVector{T}) = conj(rowvec.vec)
 @inline ctranspose{T<:Real}(rowvec::RowVector{T}) = rowvec.vec
 
 parent(rowvec::RowVector) = rowvec.vec
 
-# Strictly, these are unnecessary but will make things stabler if we introduce
-# a "view" for conj(::AbstractArray)
-@inline conj(rowvec::RowVector) = RowVector(conj(rowvec.vec))
+"""
+    conj(rowvector)
```

Comment:
not worth going through CI again right now, but this should probably say `conj(v::RowVector)` to be specific?